src/Domain/User/User.ts

```diff
@@ -45,7 +45,7 @@ export class User {
   @Column({
     name: 'server_encryption_version',
     type: 'tinyint',
-    default: 0,
+    default: User.DEFAULT_ENCRYPTION_VERSION,
```

Comment:
This is wrong, will fix later and create a task for defaulting the User encryption version to 1 on the DB side if it should be that.
@karolsojko @mobitar 
per my recent comment in https://app.asana.com/0/1199914526147392/1200004328990526/f

Reply:
[Spin-off created](https://app.asana.com/0/1200017387355215/1200151659461076)